modules/nw-ingress-gateway-api-manage-succession.adoc

```diff
@@ -0,0 +1,62 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-manage-succession_{context}"]
+= Preparing for Gateway API management succession by the Ingress Operator
+
+Starting in {product-title} 4.19, the Ingress Operator manages the lifecycle of any Gateway API custom resource definitions (CRDs). Updating from a version before 4.19 of {product-title} where this management was not present requires you to replace or remove any Gateway API CRDs that already exist in the cluster so that they conform to the specific {product-title} specification required by the Ingress Operator. {product-title} version 4.19 requires Gateway API Standard version 1.2.0 CRDs.
+
+[WARNING]
+====
+Updating or deleting Gateway API resources can result in downtime and loss of service or data. Be sure you understand how this will affect your cluster before performing the steps in this procedure. If necessary, back up any Gateway API objects in YAML format in order to restore it later.
+====
+
+.Prerequisites
+* You have installed the OpenShift CLI (`oc`).
+* You have access to an {product-title} account with cluster administrator access.
+* Optional: You have backed up any necessary Gateway API objects.
```

Comment:
Do we have any documentation on how to back up Kubernetes API objects?  Should we document an actual procedure?  

It might be as simple as an `oc get gatewayclasses.gateway.networking.k8s.io,gateways.gateway.networking.k8s.io,grpcroutes.gateway.networking.k8s.io,httproutes.gateway.networking.k8s.io,referencegrants.gateway.networking.k8s.io -o yaml > gateway-api-objects.yaml` to back up and `oc apply -f gateway-api-objects.yaml` to restore, but it might be a bit more involved.  If we don't define a procedure and test it now, we will need to do so when people start asking us what this prerequisite actually entails.

Reply:
if there is one, i'm not aware of it cc: @ahardin-rh. there isn't enough time to create a whole new one before we freeze docs for GA, so i'll add the note just underneath the prerequisites